test/studies/amr/lib/cfboundary/RefiningTransfer.chpl

```diff
@@ -294,7 +294,7 @@ class GridCFGhostSolution {
       //---- This is a zipper iteration over two objects ----
 
       for ( old_array,            current_array            ) 
-      in  zip( old_data(c_neighbor), current_data(c_neighbor) )
+      in  zip( old_data(c_neighbor)!, current_data(c_neighbor)! )
```

Comment:
More `!` on iterator expressions here. Not sure if correct or not. There are others in this file also.